resources/hg-ReferralServiceRequest.xml

```diff
@@ -1,273 +1,273 @@
+    </mapping>
+    <mapping>
         <identity value="ART-DECOR-hg-Kern" />
-        <map value="hg-dataelement-1709" />
-        <comment value="Kern" />
-      </mapping>
-    </element>
-  </differential>
+        <uri value="https://decor.nictiz.nl/pub/eerstelijnszorg/hg-html-20221201T095845/ds-2.16.840.1.113883.2.4.3.11.60.103.1.1-2020-10-19T175239.html" />
+        <name value="Huisarts- en paramedische overdrachten - Kern" />
+        <comment value="This is a preliminary mapping that should not be used for immediate implementation." />
+    </mapping>
+    <kind value="resource" />
+    <abstract value="false" />
+    <type value="ServiceRequest" />
+    <baseDefinition value="http://hl7.org/fhir/StructureDefinition/ServiceRequest" />
+    <derivation value="constraint" />
+    <differential>
+        <element id="ServiceRequest">
+            <path value="ServiceRequest" />
+            <mapping>
+                <identity value="ART-DECOR-hg-Envelop" />
+                <map value="hg-dataelement-1673" />
+                <comment value="Envelop" />
+            </mapping>
+        </element>
+        <element id="ServiceRequest.intent">
+            <path value="ServiceRequest.intent" />
+            <fixedCode value="order" />
+        </element>
+        <element id="ServiceRequest.category">
+            <path value="ServiceRequest.category" />
+            <slicing>
+                <discriminator>
+                    <type value="value" />
+                    <path value="$this" />
+                </discriminator>
+                <rules value="open" />
+            </slicing>
+            <min value="1" />
+        </element>
+        <element id="ServiceRequest.category:messageType">
+            <path value="ServiceRequest.category" />
+            <sliceName value="messageType" />
+            <short value="MessageType" />
+            <definition value="Geeft het type bericht dat verstuurd wordt door de verzender." />
+            <alias value="TypeBericht" />
+            <min value="1" />
+            <max value="1" />
+            <patternCodeableConcept>
+                <coding>
+                    <system value="urn:oid:2.16.840.1.113883.2.4.4.30.69" />
+                    <code value="1" />
+                </coding>
+            </patternCodeableConcept>
+            <mapping>
+                <identity value="ART-DECOR-hg-Envelop" />
+                <map value="hg-dataelement-1685" />
+                <comment value="TypeBericht" />
+            </mapping>
+        </element>
+        <element id="ServiceRequest.priority">
+            <path value="ServiceRequest.priority" />
+            <short value="Urgency" />
+            <definition value="Geeft de inhoud van de urgentie.&#xD;&#xA; &#xD;&#xA;In de verdere uitwerking NHG HASP-richtlijn is met het NHG bepaald dat dit een gecodeerde waardelijst kan zijn in plaats van tekst. Er is gekozen om de FHIR ValueSet RequestPriority te gebruiken." />
```

Comment:
If the description in the data set is changed (see discussion in ELZT-63), it should be changed here as well, i.e. 'In de verdere uitwerking **van de** NHG HASP-richtlijn ...'.

Reply:
Agree, we will wait on Bas' reaction in ELZT-63.

Reply:
I've chosen not to include the link to the ValueSet in the definition, since from Simplifier you can already easily click on the ValueSet itself and see the relevant codes.